src/DMXSerial2.cpp

```diff
@@ -805,6 +814,78 @@ void DMXSerialClass2::_processRDMMessage(byte CmdClass, uint16_t Parameter, bool
 
 // ADD: PARAMETER_DESCRIPTION
 
+    } else if (Parameter == SWAPINT(E120_SENSOR_DEFINITION)) { // 0x0200
+      if (CmdClass == E120_GET_COMMAND) {
+        if (_rdm.packet.DataLength != 1) {
+          // Unexpected data
+          nackReason = E120_NR_FORMAT_ERROR;
+        } else if (_rdm.packet.SubDev != 0) {
+          // No sub-devices supported
+          nackReason = E120_NR_SUB_DEVICE_OUT_OF_RANGE;
+        } else {
+          uint8_t sensorNr = _rdm.packet.Data[0];
+          if (sensorNr >= _initData->sensorsLength) {
```

Comment:
Hm... sensorsLength is defined as uint8_t, so the highest possible value would be 0xff anyway. As the "if" here checks for larger or equal, it would return E120_NR_DATA_OUT_OF_RANGE when the user would have defined 255 sensors and sensorNr is 0xff.
Maybe it's too late here (3:38 am), but I think the logic works.